CONTRIBUTING.md

```diff
@@ -75,23 +75,21 @@ How to edit OpenAPI file:
 -  If a 0-level model has dedicated sub-models, then those model's names should follow the pattern:
   `{resource}_{action}_{differentiator}_{request|response}_{body|param|query}`
   where the  `differentiator` describes the child model, e.g., `publication`.
-- If a model is used by more than one API endpoint (general model), we use simple domain language, e.g. `voucher`, `customer`, `product`, `discount`, `discount_unit`
+- If a model is used by more than one API endpoint (general model), we use simple domain language, e.g. `Customer`, `Category`, `Discount`, `Discount_unit`
 
 For example:
-- The general voucher model, used in many different API endpoints, should have the name `voucher` (currently, it has a name: `1_obj_voucher_object`)
+- The general voucher model, used in many different API endpoints, should have the name `Voucher` (currently, it has a name: `1_obj_voucher_object`)
 - for path `GET /v1/vouchers` (list vouchers), we have a `1_res_vouchers_GET` 0-level model, that should be named: `vouchers_list_response_body`.
 - for path `GET /v1/vouchers` (list vouchers), we have a `1_res_vouchers_GET` 0-level model which has sub-model `1_obj_voucher_object_list_vouchers` that should be named: `vouchers_list_item_response_body`
```

Comment:
for path `GET /v1/vouchers` (list vouchers), we have a `1_res_vouchers_GET` 0-level model which has sub-model `1_obj_voucher_object_list_vouchers` that should be named: `VouchersListItemResponseBody`

Reply:
replace `vouchers_list_item_response_body` with `VouchersListItemResponseBody`